assets/codegen.yml

```diff
@@ -33,28 +29,31 @@ generates:
       - "src/checkout/gql/queries.graphql"
     plugins:
       - "fragment-matcher"
-
   src/__generated__/schema.ts:
     plugins:
       - "typescript"
-
   src/auth/__generated__/graphql.tsx:
     documents:
       - src/auth/gql/queries.graphql
-    <<: *query_options
+    config: *query_config
```

Comment:
was this not working?

Reply:
yeah node package (probably the codegen) didn't let me do it. can't remember who it was that complained...